Sources/SKCore/BuildSystemManager.swift

```diff
@@ -96,27 +96,27 @@ extension BuildSystemManager {
   private func buildSettings(
     for document: DocumentURI,
     language: Language
-  ) async -> (buildSettings: FileBuildSettings, isFallback: Bool)? {
+  ) async -> FileBuildSettings? {
     do {
       // FIXME: (async) We should only wait `fallbackSettingsTimeout` for build
       // settings and return fallback afterwards. I am not sure yet, how best to
       // implement that with Swift concurrency.
       // For now, this should be fine because all build systems return
       // very quickly from `settings(for:language:)`.
       if let settings = try await buildSystem?.buildSettings(for: document, language: language) {
-        return (buildSettings: settings, isFallback: false)
+        return settings
       }
     } catch {
       logger.error("Getting build settings failed: \(error.forLogging)")
     }
-    if let settings = fallbackBuildSystem?.buildSettings(for: document, language: language) {
-      // If there is no build system and we only have the fallback build system,
-      // we will never get real build settings. Consider the build settings
-      // non-fallback.
-      return (buildSettings: settings, isFallback: buildSystem != nil)
-    } else {
+    // If there is no build system and we only have the fallback build system,
+    // we will never get real build settings. Consider the build settings
+    // non-fallback.
```

Comment:
This comment now applies to the `settings.isFallback = buildSystem != nil` line, which, just to make it even more readable would write as

```swift
if buildSystem == nil {
  // If there is no build system and we only have the fallback build system,
  // we will never get real build settings. Consider the build settings
  // non-fallback.
  settings.isFallback = false
}

Reply:
agree, let me update the code.